push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/jms/AbstractJMSMessageProducer.java

```diff
@@ -104,14 +103,22 @@ private void send(Destination destination, Serializable message, String property
                 connection = connectionFactory.createConnection();
             }
             Session session = connection.createSession(transacted, Session.AUTO_ACKNOWLEDGE);
-            MessageProducer messageProducer = session.createProducer(destination);
+            MessageProducer messageProducer;
```

Comment:
Making this final would ensure, even for future developer, that a value is assigned to this variable